.travis.yml

```diff
@@ -66,6 +66,19 @@ matrix:
         - DESC="package no error"
         - CMD="mvn clean package -Passembly"
 
+    # checkstyle regression no exception (oraclejdk8)
+    - jdk: oraclejdk8
+      env:
+        - DESC="checkstyle regression no exception"
+        - CMD1="git clone https://github.com/checkstyle/checkstyle &&"
+        - CMD2="cd checkstyle &&"
+        - CMD3="git checkout -b test-branch &&"
+        - CMD4="git checkout master &&"
+        - CMD5="git reset --hard HEAD^ &&"
```

Comment:
> Only the latest one or the top X? 

If we use the tops one, it seems to me that sometimes we could be testing nothing, but we should make sure we have no problems with latest commit, so I guess we can keep this.
Can we add one more with specific commits which we know will produce a configuration?